src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task{
```

Comment:
Hmm, is there any reason to not provide header comments for public classes? Also, it's easy to miss out spaces like this case `Task{`, you might want to set up an actual code formatter or say intellij to auto format on save to avoid these 😄